### ISSUE TITLE
Include in the Vim runtime distribution

### ISSUE THREAD

**Issue by dkearns** [rodjek/vim-puppet#131]
It would be nice if this project could be included in the Vim runtime distribution.

It's a shame not to have it available in default installs for the casual Puppeteer.

**Comment by lelutin** [comment]
it sounds interesting indeed. I guess to achieve that the license would need to be changed to Vim's license though, am I right?

**Comment by dkearns** [comment]
I don't believe so.  There's other runtime files distributed under the Apache license.

E.g., https://groups.google.com/g/vim_dev/c/gWc-1C96AXM/m/xELX9046BAAJ
